openapi.yaml

```diff
@@ -1107,6 +1155,26 @@ components:
                 transaction_id:
                   type: string
                   description: stellar identifier of the blockchain transaction
+    JWTBodyMigrationList:
+      description: "Add users to the gradual blockchain migration"
+      allOf:
+        - $ref: '#/components/schemas/JWTBodyCommon'
+        - type: object
+          required:
+            - user_ids
+          properties:
+            sub:
+              type: string
+              enum:
+                - migrate_users
+            user_ids:
+              type: array
+              items:
+                type: string
+                description: a user_id to be migrated
```

Comment:
app_user_id

Reply:
To the outside we call app_user_ids user_ids. Because user_ids are internal.